src/components/app/Root.js

```diff
@@ -233,7 +233,7 @@ const options: ExplicitConnectOptions<
   },
   component: ProfileViewWhenReadyImpl,
 };
-const ProfileViewWhenReady = explicitConnect(options);
+export const ProfileViewWhenReady = explicitConnect(options);
```

Comment:
I've seen this but because it doesn't happen when running all the tests I didn't invest time to look at it. But I'll do :)

Reply:
> (node:26860) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'open' of undefined

This is https://github.com/devtools-html/perf.html/blob/3d71e2cc7c7f074a7d1787bdb25c05abc0b22cfa/src/profile-logic/symbol-store-db.js#L95

> (node:26860) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'getProfile' of undefined
this comes from https://github.com/devtools-html/perf.html/blob/3d71e2cc7c7f074a7d1787bdb25c05abc0b22cfa/src/actions/receive-profile.js#L334

I don't know yet if my approach will be mocking these objects, or if I should implement something closer to the reality in the tests (eg: instead of dispatching the errors / viewProfile / etc in tests, let the actual code do that -- even if we already have tests handling this).

Reply:
I decided to mock this whole module. Now we don't have the error anymore!